package.json

```diff
@@ -62,7 +64,7 @@
     },
     "particle-dev-profiles": {
       "versions": {
-        "^0.0.1": "consumeProfiles"
+        "^0.0.2": "consumeProfiles"
```

Comment:
Is the provided interface version bumped also? I see it still being at `0.0.1`: https://github.com/spark/particle-dev-profiles/blob/feature/metrics/package.json#L32